components/dashboards-web-component/src/PagesNavigationPanel.jsx

```diff
@@ -43,8 +47,42 @@ class PagesNavigationPanel extends React.Component {
                     {this.props.pagesList}
                 </ul>
             </div>
+            <hr/>
+            <MuiThemeProvider>
```

Comment:
Typo? Shouldn't it be MultiThemeProvider

Reply:
No. I think it is for "material ui provider".